src/server/firefox/firefox.ts

```diff
@@ -31,6 +32,18 @@ export class Firefox extends BrowserType {
     super('firefox', playwrightOptions);
   }
 
+  executablePath(options?: types.LaunchOptions): string {
+    if (options?.channel) {
+      let executablePath = undefined;
+      if ((options.channel as any) === 'stable')
```

Comment:
Instead of calling it stable, let's call it maybe `firefox-stable` and also add it to the `BrowserChannel` type?

Reply:
Agreed on `firefox-stable`. Will probably postpone adding to `BrowserChannel` type for now to avoid documenting.